app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -292,7 +292,9 @@ export default {
       this.setScrollParams();
 
       const dataFetchCheck =
-        this.getMessages.dataFetched === true && this.shouldLoadMoreChats;
+        this.getMessages &&
+        this.getMessages.dataFetched === true &&
+        this.shouldLoadMoreChats;
```

Comment:
I wouldn't go with this fix as this just suppresses the error and the actual problem is not solved.

The problem here I guess is that this view is getting render even when a conversation is not available.